src/python/library/setup.py

```diff
@@ -82,7 +82,7 @@
 data_files = [
     ("", ["LICENSE.txt"]),
 ]
-if (PLATFORM_FLAG != "any") and ("@TRITON_ENABLE_PERF_ANALYZER@" == "ON"):
+if (PLATFORM_FLAG != "any") and ("@TRITON_PACKAGE_PERF_ANALYZER@" == "ON"):
```

Comment:
Is this an issue? If not, could you dismiss it with an explanation?

Reply:
I think this is a non-issue. Essentially the issue being repoted is saying: 
```python
# BAD
def is_the_answer() -> bool:
    return 42 == 42  # [comparison-of-constants]

# GOOD
def is_the_answer(meaning_of_life: int) -> bool:
    return meaning_of_life == 42
```
In our case what's happening is we're executing `configure_file(setup.py setup.py @ONLY)` from [here](https://github.com/triton-inference-server/client/blob/main/src/python/library/CMakeLists.txt#L48C1-L48C40), which will replace `@TRITON_PACKAGE_PERF_ANALYZER@` with either "ON" or "OFF". So we're left with a comparison like:
```python
"ON" == "ON"
```
If I am understanding the warning correctly, I believe it wants us to treat the presence of `TRITON_PACKAGE_PERF_ANALYZER` as a boolean instead, e.g.,:
```python
if(TRITON_PACKAGE_PERF_ANALYZER):
    ....
```
However, this is not possible for our case because if `TRITON_PACKAGE_PERF_ANALYZER` was set to "OFF" our if-condition would still evaluate to True.